webpack.playground.config.js

```diff
@@ -69,11 +70,40 @@ module.exports = {
     port: PORT,
     open: true,
     before (app) {
+
       app.get('/app/UserHome', (req, res) => {
         res.status(200)
           .send('<h1>Mock User Dashboard</h1><a href="/">Back to Login</a>');
       });
 
+      const bodyParser = require('body-parser');
```

Comment:
can be deleted now?

Reply:
Should we just get rid of `/playground/responseConfig` API itself ?

Reply:
yes!